examples/integrations/micronaut/data/src/main/java/io/helidon/examples/integrations/micronaut/data/BeanValidationExceptionMapper.java

```diff
@@ -16,7 +16,8 @@
 
 package io.helidon.examples.integrations.micronaut.data;
 
-import jakarta.validation.ConstraintViolationException;
+import javax.validation.ConstraintViolationException;
```

Comment:
Should this actually be changed from `jakarta` to `javax`? 

Same question for the similar change in `PetResource.java` below.

Reply:
With the current version of `micronaut`, it does not handle `jakarta.validation`.

Here is the PR : 
Revert back to using javax.validation as jakarta.validation is not supported by micronaut [#4476 ](https://github.com/helidon-io/helidon/pull/4476)